apps/allocations/app/components/App/App.js

```diff
@@ -108,26 +165,71 @@ const App = () => {
     children: PropTypes.node.isRequired,
   }
 
-  if (accounts.length === 0) {
+  if (budgets.length === 0) {
     return <Wrap><Empty action={onNewBudget} /></Wrap>
   }
 
   return (
-    // TODO: Profile App with React.StrictMode, perf and why-did-you-update, apply memoization
     <Wrap>
       <Header
         primary="Allocations"
         secondary={
           <Button mode="strong" icon={<IconPlus />} onClick={onNewBudget} label="New budget" />
         }
       />
-      <Accounts
-        accounts={accounts}
+      <Budgets
+        budgets={budgets}
         onNewAllocation={onNewAllocation}
+        onEdit={onEdit}
+        onDeactivate={onDeactivate}
+        onReactivate={onReactivate}
+      />
+      <AllocationsHistory allocations={allocations} />
+      <DeactivateModal
+        state={modal}
+        onClose={closeModal}
+        onSubmit={onSubmitDeactivate}
       />
     </Wrap>
   )
 }
 
-// eslint-disable-next-line import/no-unused-modules
+const DeactivateModal = ({ state, onClose, onSubmit }) => {
+  const deactivate = () => {
+    onSubmit(state.budgetId)
+  }
+  return (
+    <Modal visible={state.visible} onClose={onClose}>
+      <div css={{ fontSize: '26px' }}>
+        Deactivate budget
+      </div>
+      <div css={{ marginTop: '32px' }}>
+        Deactivating this budget will immediately disable it once the decision is enacted. You may choose to reactivate this budget at any time.
+      </div>
+      <div css={{
```

Comment:
Using divs for everything does not look really nice, and defeats the semantic web purposes. Try to define Styled components (we use this heavy library for some reason), if possible making them reusable, or rely instead into `Text` components from `aragon-ui` if you can (like the case here)

If the Aragon UI design system changes while we use their `Text` components, we don't need to do anything about it, but if we abuse `div`s and `css={{}}` everywhere then we need to invest lot of time maintaining and "modernizing" our apps just for a single css change in the aragon design system, please adopt the good practice now before it is too late.

Also makes our layouts difficult to read an maintain.

Also it is a wide accepted convention among web development industry to avoid using plain `div`s  for everything and it is against the World Wide Web modern standards